drivers/usb/device/usb_dc_native_posix_adapt.h

```diff
@@ -67,6 +67,11 @@ struct usbip_submit {
 	int32_t start_frame;
 	int32_t number_of_packets;
 	int32_t interval;
+	uint8_t bmRequestType;
```

Comment:
Why Hungarian notation and camel case all of sudden?

Reply:
In USB we (also Linux Kernel) use it for the identifiers that come from Chapter 9 of the USB Specification.

Reply:
It's a name defined in the USB core spec